python_modules/dagster-test/dagster_test/components/simple_asset.py

```diff
@@ -2,21 +2,19 @@
 from dagster._core.definitions.decorators.asset_decorator import asset
 from dagster._core.definitions.definitions_class import Definitions
 from dagster._core.execution.context.asset_execution_context import AssetExecutionContext
-from dagster_components import Component, ComponentLoadContext
-from pydantic import BaseModel
+from dagster_components import Component, ComponentLoadContext, ResolvableModel, ResolvedFrom
 
 
-class SimpleAssetSchema(BaseModel):
+class SimpleAssetComponentModel(ResolvableModel):
```

Comment:
we don't need this right? we can just make SimpleAssetComponent a ResolvableModel